attestation/commandrun/tracing_linux.go

```diff
@@ -46,12 +50,6 @@ type ptraceContext struct {
 	environmentBlockList map[string]struct{}
 }
 
-func enableTracing(c *exec.Cmd) {
```

Comment:
as above, enabling tracing is fairly platform dependent -- linux is the only platform that supports ptrace so this needs to be in the linux gated file